src/reactDOMRe.re

```diff
@@ -1,13 +1,13 @@
-external render : ReactRe.reactElement => ReasonJs.Document.element => unit =
+external render : ReactRe.reactElement => ReasonJs.Dom.element => unit =
   "render" [@@bs.val] [@@bs.module "react-dom"];
 
-external unmountComponentAtNode : ReasonJs.Document.element => unit =
+external unmountComponentAtNode : ReasonJs.Dom.element => unit =
   "unmountComponentAtNode" [@@bs.val] [@@bs.module "react-dom"];
 
-external findDOMNode : ReactRe.reactRef => ReasonJs.Document.element =
+external findDOMNode : ReactRe.reactRef => ReasonJs.Dom.element =
   "findDOMNode" [@@bs.val] [@@bs.module "react-dom"];
 
-external domElementToObj : ReasonJs.Document.element => Js.t {..} = "%identity";
+external domElementToObj : ReasonJs.Dom.element => Js.t {..} = "%identity";
```

Comment:
You'll want all these functions to take `ReasonJs.Dom.element_like _`. That way they'll accept any "subtype" of element too.